app/views/mou/index.html.erb

```diff
@@ -1,3 +1,5 @@
+<% mou_attached = @current_org_signed_mou %>
```

Comment:
Don't see this intermediate variable being needed

Reply:
Good spot